docs/deploy-full-rp-service-in-dev.md

```diff
@@ -0,0 +1,108 @@
+
+    export ARO_IMAGE=${DST_ACR_NAME}.azurecr.io/aro:$(git rev-parse --short HEAD)-dirty
+    ```
+
+1. Run `make deploy`
+> __NOTE:__ This will fail on the first attempt to run due to certificate and container mirroring requirements.
+
+1. Update the certificates in keyvault
+    ```bash
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name rp-mdm \
+        --file secrets/rp-metrics-int.pem >/dev/null
+   az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name rp-mdsd \
+        --file secrets/rp-logging-int.pem >/dev/null
+   az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name cluster-mdsd \
+        --file secrets/cluster-logging-int.pem >/dev/null
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name dev-arm \
+        --file secrets/arm.pem >/dev/null
+   az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name rp-firstparty \
+        --file secrets/firstparty.pem
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name rp-server \
+        --file secrets/localhost.pem >/dev/null
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-por" \
+        --name portal-server \
+        --file secrets/localhost.pem >/dev/null
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-por" \
+        --name portal-client \
+        --file secrets/portal-client.pem >/dev/null
+    ```
+
+1. Mirror the OpenShift images to your new ACR
+
+    1. Setup mirroring environment variables
+        ```bash
+        export SRC_AUTH_QUAY=FILL_IN # Get quay auth https://cloud.redhat.com/openshift/create/local -> Download Pull SEcret
+        export SRC_AUTH_REDHAT=$(echo $USER_PULL_SECRET | jq -r '.auths."registry.redhat.io".auth')
+        export DST_AUTH=$(echo -n '00000000-0000-0000-0000-000000000000:'$(az acr login -n ${DST_ACR_NAME} --expose-token | jq -r .accessToken) | base64 -w0)
+
+    1. Get the installer image needed to mirror.  This is located in `pkg/util/version/const.go` under the `InstallStream PullSpec` struct.  Should be formatted like `quay.io/openshift-release-dev/ocp-release@sha256:a7b23f38d1e5be975a6b516739689673011bdfa59a7158dc6ca36cefae169c18`
+
+    1. Run the mirroring
+        ```bash
+        go run ./cmd/aro mirror quay.io/openshift-release-dev/ocp-release@sha256:a7b23f38d1e5be975a6b516739689673011bdfa59a7158dc6ca36cefae169c18
+        ```
+
+1. Push the ARO image to your ACR
+    1. Build and push the ARO image
+        ```bash
+        make publish-image-aro-multistage
+        ```
+
+1. Run `make deploy`
+
+
+## Deploying a cluster
+
+1. Setup a local tunnel to the RP
+    ```bash
+    go run ./hack/tunnel $(az network public-ip show -g ${RP_RESOURCE_GROUP} -n rp-pip | jq -r '.ipAddress')
```

Comment:
nice! -> Makefile target maybe?

Reply:
can we facilitate people SSHing to the master VM as well?
1) currently need to update firewall rules to enable ssh
2) a documented command / script / Makefile target to ssh?

Reply:
[Added a section in the readme for this](https://github.com/Azure/ARO-RP/pull/1406/files#diff-c86a95d65bd0a05430440eea9a71536a261fcebfdfddd637e681bb00e9eebbcaR147)  It's a bit messy so I don't think it should be a Makefile target.